packages/edit-site/src/components/global-styles/block-preview-panel.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { BlockPreview } from '@wordpress/block-editor';
+import { getBlockType, getBlockFromExample } from '@wordpress/blocks';
+import { useResizeObserver } from '@wordpress/compose';
+
+// const normalizedWidth = 248;
+const normalizedHeight = 152;
+
+const BlockPreviewPanel = ( { name } ) => {
+	const blockExample = getBlockType( name )?.example;
+	const panelStyles = {
+		width: '100%',
+		height: `${ normalizedHeight }px`,
+		overflow: 'auto',
+	};
+	const [ containerResizeListener, { width: containerWidth } ] =
+		useResizeObserver();
+
+	return ! blockExample ? null : (
+		<div
+			className="edit-site-block-styles-preview__panel"
```

Comment:
This class name isn't following BEM properly. It should be `edit-site-global-styles__block-preview-panel`.